test/commentupdate.spec.js

```diff
@@ -0,0 +1,142 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../server/app';
+
+chai.use(chaiHttp);
+const { should } = chai;
+should();
+
+let token;
+let tokenJohn;
+
+describe('Tests for Update Comments', () => {
+  describe('update comment tests', () => {
+    before((done) => {
+      chai
+        .request(app)
+        .post('/api/v1/users/auth/login')
+        .send({
+          email: 'postman@gmail.com',
+          password: 'Password'
+        })
+        .end((err, res) => {
+          token = res.body.data.token;
+          res.body.should.have.property('data');
+          done();
+        });
+    });
+    before((done) => {
+      chai
+        .request(app)
+        .post('/api/v1/users/auth/login')
+        .send({
+          email: 'johndoe@gmail.com',
+          password: 'paSS1234'
+        })
+        .end((err, res) => {
+          tokenJohn = res.body.data.token;
+          res.body.should.have.property('data');
+          done();
+        });
+    });
+
+    it('should fail if commentId is invalid', (done) => {
+      chai
+        .request(app)
+        .put('/api/v1/articles/1/comments/err')
+        .set('Authorization', token)
+        .send({
+          content: 'Your post was not inspiring.'
+        })
+        .end((err, res) => {
+          res.body.status.should.equal('fail');
+          res.body.data.message.should.equal('Invalid commentId');
+          done();
+        });
+    });
+    it('should fail if commentId is invalid', (done) => {
```

Comment:
Nice Work James  👍. You have a duplicated test description for two different tests on line 43 and line 57.

Reply:
You also have a same description for two tests on line 99 and 113.

Reply:
That's an eagle eye look Tomi. Thank you for noticing. I would make changes to it in my next branch.